ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -48,6 +48,8 @@ class JITModBuilder {
   Instruction *StartTracingInstr = nullptr;
 
 public:
+  // Store virtual addresses for called functions.
+  std::map<StringRef, uint64_t> globalMappings;
```

Comment:
Should that be `GlobalMappings`, with a capital `G`?

Reply:
Under protest, yes. I just can't get used to this weird way of naming things. We should have never adopted LLVM's way here.